src/main/java/pl/javastart/task/Ticket.java

```diff
@@ -0,0 +1,62 @@
+package pl.javastart.task;
+
+public class Ticket {
+
+    private String ticketName;
+    private String ticketPlace;
+    private String typeOfTicket;
+    private int ticketPrice;
+    private int ticketNumber;
+
+    public Ticket(String ticketName, String ticketPlace, String typeOfTicket, int ticketPrice, int ticketNumber) {
+        this.ticketName = ticketName;
+        this.ticketPlace = ticketPlace;
+        this.typeOfTicket = typeOfTicket;
+        this.ticketPrice = ticketPrice;
+        this.ticketNumber = ticketNumber;
```

Comment:
Jeżeli w metodzie main utworzysz kilka obiektów reprezentujących bilety, to każdy z nich powinien mieć kolejne identyfikatory, czyli 1, 2, 3. Nie powinno być konieczności samodzielnego przekazywania identyfikatora jako argumentu konstruktora.
Dla inspiracji https://app.javastart.pl/kurs/java/java-podstawy-temat-16-skladowe-statyczne/cwiczenie/java-static-cwiczenie-1